tests/integration/devfile/cmd_devfile_env_test.go

```diff
@@ -81,26 +81,6 @@ var _ = Describe("odo devfile env command tests", func() {
 				testDebugPort,
 			}
 			helper.MatchAllInOutput(output, wantOutput)
-		})
-
-		It("Should fail to set the invalid parameter", func() {
```

Comment:
Its good to keep a negative case for validation.

Reply:
Its already there https://github.com/openshift/odo/pull/3793/files#diff-e3308315be7824d6d6b338513357491bR94

Reply:
Ahh...found it
https://github.com/openshift/odo/pull/3793/files#diff-e3308315be7824d6d6b338513357491bR96